samples/dotnet-isolated/confluent/local.settings.json.example

```diff
@@ -0,0 +1,10 @@
+{
+    "IsEncrypted": false,
+    "Values": {
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+        "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
+        "BrokerList": "{YOUR_CONFLUENT_CLOUD_BROKER}.eastus.azure.confluent.cloud:9092",
```

Comment:
Should we change this to {YOUR_CONFLUENT_CLOUD_BROKER}

Reply:
Done